en/services/component_information.md

```diff
@@ -179,18 +180,78 @@ Specifically, the following information is provided:
 
 A GCS can provide a UI for testing outputs based on the configured output functions, by iterating over all output channels and collecting the configured actuator output functions, and then utilizing the `MAV_CMD_ACTUATOR_TEST` command.
 
+## Translation
 
-## Open Issues
+At high-level, metadata translation works as follows:
+
+- The metadata provider sets the `translationUri` in [general metadata file](#COMP_METADATA_TYPE_GENERAL) for each metadata type that supports translation.
+  There is no CRC for the resource pointed to by this URI, as translations might change independently of metadata (for example, adding or changing translations for a particular language, or adding translations for a new language).
+- The `translationUri` URL points to a summary JSON file that contains links to the separate files that contain each translation of the particular metadata type.
+  The translation summary JSON file also contains modification timestamps for each linked translation file so that a GCS can determine whether the file has been updated.
+  The translation files are a (compressed) file in [TS file format](https://doc.qt.io/qt-6/linguist-ts-file-format.html).
+- A client (GCS) downloads the summary file then uses its contained to locate and download the translation file(s) it is interested in.
+- The client can then apply the downloaded translations to the metadata json file(s) (which contains annotations for which tags to translate) using the translation TS.
+
+### Caching
+
+The following caching strategy is recommended for clients:
+
+- Locally cache the downloaded translation files.
+  These should be used until successfully replaced with a newer version.
+- After 3 days attempt to download the summary JSON file again.
```

Comment:
I assume you actually do the check when you start QGC. So the actual update rate is 3 days + when you start GCS and it happens to have connectivity? Is that right? If so, perhaps we should note that.

Reply:
It's upon vehicle connection initialization, when receiving the metadata.

> For example what happens if you connect to the vehicle but there is no internet connection initially - do you cache that the vehicle connected and add that to your download list for later?

No, but that's a possible extension.